### ISSUE TITLE
DB Upload, VOCABULARY

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/CohortDiagnostics#714]
I have a DP that has Common Classification of Medical Acts (CCAM) in their SOURCE_TO_CONCEPT_MAP table as it didn't exist in the OMOP Vocabulary at the time.  In the CD the data came across like this:

| "vocabulary_id" | "vocabulary_name" | "vocabulary_reference" | "vocabulary_version" | "vocabulary_concept_id" |
|-----------------|-------------------|------------------------|----------------------|-------------------------|
| "CCAM"          | "CCAM"            | "AP-HM vocabulary"     |                      | 0                       |

However, now CCAM does exist in the VOCAB.  When other DBs go to load the CCAM DB in the VOCABULARY table with CohortDiagnostics::uploadResults() I get an error like this as the other DBs have

~~~
java.sql.BatchUpdateException: Batch entry 3 
INSERT INTO covid_aesi.vocabulary (vocabulary_id,vocabulary_name,vocabulary_reference,vocabulary_version,vocabulary_concept_id) 
VALUES('CCAM','Common Classification of Medical Acts','https://www.ameli.fr/accueil-de-la-ccam/telechargement/index.php','CCAM version 64',32957) 
was aborted: ERROR: duplicate key value violates unique constraint "vocabulary_pkey"
~~~

**Comment by gowthamrao** [comment]
This enhancement would not allow duplication in primary key in the cohort diagnostics output
https://github.com/OHDSI/CohortDiagnostics/pull/748

**Comment by azimov** [comment]
I have attempted to fix this problem in #762. We either need to allow this somehow or force DPs to obey our database constraints.

**Comment by azimov** [comment]
Patch has been merged in the develop branch
